### DISCUSSION TITLE
How to load .Robj file

### DISCUSSION THREAD

**Discussion by Blatta-Garyx** [satijalab/seurat#4021] [Q&A] [answered]
Hello, a very basic question. How do I load the Seurat  .Robj files downloaded from Figshare. Thanks. Rishi.

**Comment by timoast** [comment, marked as answer]
You can use the `load()` function in R to load objects that were saved using the `save()` function, or use `readRDS()` for those saved using `saveRDS()`.

**Reply by Blatta-Garyx** [reply]
Thank you. The load(). Worked. RR.

**Comment by strawberry098** [comment]
After loading the .Robj file (ref) downloaded from Figshare, I want to integrate it with an .rds file (slide.seq) for label transfer:

`anchors <- [FindTransferAnchors](https://satijalab.org/seurat/reference/FindTransferAnchors.html)(reference = ref, query = slide.seq, normalization.method = "SCT",
    npcs = 50)`

but I get the following error:

Error in UseMethod(generic = "DefaultAssay", object = object) : 
  no applicable method for 'DefaultAssay' applied to an object of class "seurat"
